en/docs/reference/pricing.md

```diff
@@ -2,229 +2,137 @@
 
-#### Scenario 
- 
-Airlines compute ticket prices typically once per day. The ticket prices depend on a number of factors. The price of oil is one such major factor that affects the pricing.  
 
-The [scheduler](#scheduler) on Choreo runs once per day and fetches the oil prices for the day from the [Oil Price API](#oilPriceAPI). It then computes the jet fuel price applying the price calculation formula. The scheduler then updates the daily jet fuel price in the [Air Fare Database](#airFareDatabase).
+#### Monitoring and observability
 
-Having the jet fuel price, the scheduler is now able to calculate the airfare for each route. The Scheduler retrieves the airline routes from the [Airline Routes API](#airlineRoutesAPI). Upon receiving the airline routes, it iterates over each route, computes the airfare, and updates the AirFare database.
+Choreo supports monitoring and observability features for each plan as follows:
 
-#### Glossary 
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/><br/>Alerting | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties |
 
-- <a class="pricing" id="scheduler">Scheduler</a>: A scheduled task created on Choreo that runs every day.
-- <a class="pricing" id="oilPriceAPI">Oil Price API</a>: An external-facing API that returns the crude oil price for the day.
-- <a class="pricing" id="airFareDatabase">Air Fare Database</a>: The database that stores the jet fuel price which is used to compute the airfare for each route.
-- <a class="pricing" id="airlineRoutesAPI">Airline Routes API</a>: An external-facing API that provides details about the airline routes and a list of airline routes.
-- <a class="pricing" id="airlineRoutesDatabase">Airline Routes Database</a>: The database that stores the airline routes information.
+The maximum time interval for which users on each plan can view logs, traces and alerts is 30 days.
 
-<br/>
+#### Security
 
-#### Update jet fuel price daily
+Choreo supports the following security features for all the plans:
 
-![ Update jet fuel prices daily](../assets/img/references/scheduled-task-example1.png)
+- SSO for API Developer Portal
+- Enterprise login
+- Container manifest scanning
+- Container static scanning
+- Run-time container security
 
-Let’s see how we can calculate the steps for the update maintenance process:
+#### Cloud-code editing
```

Comment:
@RukshiW we can remove this section. Lets keep a backup and remove.

Reply:
Done